src/config.test.ts

```diff
@@ -1,6 +1,6 @@
 import fs from 'fs';
 import { generateRandomAddress } from '../test/test-utils';
-import loadConfig from './config';
+import { loadConfig } from './config';
 
 jest.mock('fs');
```

Comment:
Maybe we can add one to test functionality of validateConfig when `ignoreInterpolationStrings` params set to true.

Reply:
Yes, I first wanted to validate the idea behind exposing this functionality and to know if you agree with even being able to validate a config.json object for without interpolating its values first. Since it seems this might be a good idea then I'll add a test 💪🏻